### DISCUSSION TITLE
unusually stubborn queue on builds with Mac mini M1?

### DISCUSSION THREAD

**Discussion by JavascriptMick** [codemagic-ci-cd/codemagic-docs#2368] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. configure build (flutter, mac m1 mini)
2. run build
3. build stuck in enqueued

### Expected results

build runs or fails

### Actual results

build stuck in enqueued

### Build id (optional)

65c7396fd512b4c674053a92

**Comment by mikhail-tokarev** [comment]
Hey! We have identified another pool of malicious users who consumed our machine resources and blocked them. All builds should start quickly. 

Sorry for the inconvenience!
